docs/tips/03-if-else-in-JSX.md

```diff
@@ -33,23 +33,23 @@ That's not valid JS. You probably want to make use of a ternary expression:
 React.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
 ```
 
-If a ternary expression isn't robust enough, you can use `if` statements to determine which
-components should be used.
+You can also use ternary expression to determine which tags or components should be used.
```

Comment:
A few things.

1. They're all components
2. I'd really like to keep an `if/else` case in here just so people can see this pattern. Ternaries get super ugly beyond the simple, self-closing case. So let's talk about both in the text here, and have both examples.
3. I don't like your formatting in the code block so I'll suggest something else (though I think this should stay an if/else and leave the other one as a ternary.
4. cc @chenglou who might have some other thoughts on this page


```js
var loginInformation = isLoggedIn ?
  <cite>Hello!</cite> :
  <strong>Please log in</strong>;
```